src/ui/views/ConfiguratorView/index.tsx

```diff
@@ -677,27 +677,48 @@ const ConfiguratorView: FunctionComponent<ConfiguratorViewProps> = (props) => {
       if (dnsDevice) {
         const targetName = dnsDevice.target.toUpperCase();
 
-        const device = deviceTargets?.find((item) => {
+        // if targetName is an empty string, then return
+        if (targetName.trim().length === 0) {
+          return;
```

Comment:
What will happen when user select this target? Nothing? If so, I think feedback should be visible to the user somehow. For example an error message could be displayed, same thing for other cases where you just return from the callback.

Reply:
Another option for case where target is just empty - "Select" button could be hidden for example